examples/plots/Skew-T_Layout.py

```diff
@@ -41,16 +41,12 @@
 skew.plot(p, T, 'r')
 skew.plot(p, Td, 'g')
 skew.plot_barbs(p, u, v)
-skew.ax.set_ylim(1000, 100)
 
 # Add the relevant special lines
 skew.plot_dry_adiabats()
 skew.plot_moist_adiabats()
 skew.plot_mixing_lines()
 
-# Good bounds for aspect ratio
-skew.ax.set_xlim(-30, 40)
```

Comment:
I'm going to say no, no it doesn't. At least not until we figure out the problems with aspect ratio.